extensions/mssql/src/objectManagement/ui/databaseRoleDialog.ts

```diff
@@ -70,19 +73,15 @@ export class DatabaseRoleDialog extends ObjectManagementDialogBase<ObjectManagem
 	}
 
 	private initializeMemberSection(): void {
-		this.memberTable = this.createTable(localizedConstants.MemberSectionHeader, [
-			{
-				type: azdata.ColumnType.text,
-				value: localizedConstants.NameText
-			}
-		], this.objectInfo.members.map(m => [m]));
+		this.memberTable = this.createTable(localizedConstants.MemberSectionHeader, [localizedConstants.NameText], this.objectInfo.members.map(m => [m]));
 		const buttonContainer = this.addButtonsForTable(this.memberTable, localizedConstants.AddMemberAriaLabel, localizedConstants.RemoveMemberAriaLabel,
 			async () => {
 				const dialog = new FindObjectDialog(this.objectManagementService, {
 					objectTypes: [ObjectManagement.NodeType.DatabaseRole, ObjectManagement.NodeType.User],
```

Comment:
I see `objectTypes` now has type `string[]` instead of `NodeType[]`, this makes sense for the principal search dialog as it can have a lot of returned objectTypes. But this line is still using `NodeType.DatabaseRole` and `NodeType.User`, I think I'll add a convert func in STS to convert the type string `SearchableType` when it looks like `NodeType`.

Reply:
if you look at the js file compiled from here, they are all strings.

also in C#, You don't have to do the conversion, the json deserialization will take care of it. 
![image](https://user-images.githubusercontent.com/13777222/236518926-57ee37cf-7032-4c4c-aee0-1e10078fefc3.png)

Reply:
By conversion I mean from SqlObjectType string to SearchableObjectType.
For principal dialog, there's no such problem because all the options are read from STS. 
For database role dialog, the options are hard-coded on the front end.
I can figure this part out.

Reply:
if you end up needing to map multiple values from to one Enum you can refer to this post: https://stackoverflow.com/questions/58558186/how-do-i-add-multiple-values-in-enummember-attribute